### ISSUE TITLE
Configuration setting to disable upload prompt

### ISSUE THREAD

**Issue by zachriggle** [faressoft/terminalizer#159]
It would be nice if there were a config setting that would disable the following prompt:

```
The command `terminalizer share`can be used anytime to share recordings!
? Would you like to share your recording on terminalizer.com? (Y/n)
```

So that I do not accidentally upload recordings while doing things at work.

**Comment by ognjenjevremovic** [comment]
Hey @zachriggle, I just looked at the code and it seems that there is a flag for this.

```sh
-k, --skip-sharing  Skip sharing and showing the sharing prompt message [boolean] [default: false]
```

This flag, if present, will remove the prompt of sharing the recordings. However, this flag will not affect the first message (which will always be printed out):
```sh
The command `terminalizer share`can be used anytime to share recordings!
```

I'll be glad to provide a PR and address this (that the actual flag addresses both messages), if that of course aligns with the specification of the tool. I see the first message more as an info (about the tool feature). 
However, I will need someone from the core / maintainers team to confirm that this is the desired behavior (that the `--skip-sharin` should remove both messages from the log), prior to submitting the PR myself.

**Comment by rosskusler** [comment]
I think it would be convenient if we could include some setting in the config file so that we don't have to remember to specify -k when using this tool.

**Comment by diksown** [comment]
I agree with @rosskusler, if you miss a key it can lead to uploading sensitive information.

**Comment by markjreed** [comment]
is there still no config setting for this? I have terminalizer aliased to terminalizer -k for now...
